src/modules/tests/tests.effects.ts

```diff
@@ -173,6 +173,16 @@ export class TestsEffects {
           staffNumber: employeeId,
         };
         slot = getDelegatedBookedTestSlot(delegatedRekeySearch);
+
+        // TODO remove this temporary hard keyed test centre when real data is available
```

Comment:
Could you please reference MES-5790 as the story that will fix this and remove the todo?

Reply:
added